src/main/java/io/github/thebusybiscuit/sensibletoolbox/items/WateringCan.java

```diff
@@ -124,47 +123,40 @@ public void onInteractItem(PlayerInteractEvent event) {
                 neighbour.setType(Material.AIR);
                 setWaterLevel(MAX_LEVEL);
                 newStack = toItemStack();
-            }
-            else if (STBUtil.isCrop(block.getType())) {
+            } else if (STBUtil.isCrop(block.getType())) {
                 // attempt to grow the crops in a 3x3 area, and use some water from the can
                 waterCrops(player, block);
                 waterSoil(player, block.getRelative(BlockFace.DOWN));
                 newStack = toItemStack();
-            }
-            else if (block.getType() == Material.FARMLAND) {
+            } else if (block.getType() == Material.FARMLAND) {
                 if (STBUtil.isCrop(block.getRelative(BlockFace.UP).getType())) {
                     waterCrops(player, block.getRelative(BlockFace.UP));
                     waterSoil(player, block);
                     newStack = toItemStack();
-                }
-                else {
+                } else {
                     // make the soil wetter if possible
                     waterSoil(player, block);
                     newStack = toItemStack();
                 }
-            }
-            else if (block.getType() == Material.COBBLESTONE && getWaterLevel() >= 10) {
+            } else if (block.getType() == Material.COBBLESTONE && getWaterLevel() >= 10) {
                 if (ThreadLocalRandom.current().nextBoolean()) {
                     block.setType(Material.MOSSY_COBBLESTONE);
                 }
 
                 useSomeWater(player, block, 10);
                 newStack = toItemStack();
-            }
-            else if (block.getType() == Material.STONE_BRICKS && getWaterLevel() >= 10) {
+            } else if (block.getType() == Material.STONE_BRICKS && getWaterLevel() >= 10) {
```

Comment:
Formatting :NotLikeThis: Put the `else if` on new lines please like it's done everywhere in STB.